site/src/@ndlib/gatsby-theme-marble/components/Layout/PageWrapper/Footer/index.js

```diff
@@ -14,66 +14,62 @@ export const Footer = () => {
   return (
```

Comment:
The line before code review starts contains an unused variable. Can we get this line removed?
```
  const date = new Date()
```